docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
 
 For detailed instructions on each step, follow the links above.
 
-## How do I get access to the private channel with Icebreaker documentation?
+### What endpoints do I need to open in my firewall for the Icebreaker node?
 
-After receiving your Icebreaker NFT:
+To function properly, your Icebreaker node requires the following ports to be open in your firewall:
 
-- Notify the Blockfrost team that you've received your NFT
-- A team member will add you to the private Icebreakers channel
-- Access the comprehensive documentation in the announcement channel
-- Follow the setup instructions provided there.
+- `3000/tcp` (or custom port): For incoming API requests from Blockfrost
+- `8081/tcp`: Prometheus metrics endpoint for monitoring (optional but recommended)
 
-If you've received your NFT but don't have channel access yet, post a message on [Discord](https://discord.com/invite/inputoutput) in the `#ask-blockfrost` channel for assistance.
+Ensure these ports allow incoming connections from the Blockfrost server IPs documented [here](/usage/network#firewall).
 
-## Do I need confirmation after receiving the NFT to proceed?
+You can verify port accessibility using:
+```bash
+sudo ufw allow 3000/tcp
```

Comment:
I think this is a very limited example which is mainly limited to Ubuntu Linux with ufw firewall. And still, doesn't guarantee accessibility from the external world.
I would use such tools like https://portchecker.co/ but I am not sure if we want to mention these 3rd party websites in our .
In that case, trying using another computer outside of the server network would be the best:
`telnet example.com 3000 ` or curl -v `http://YOUR_PUBLIC_IP:3000/`

Reply:
Great, added those details.